apps/backoffice-v2/src/domains/business-reports/hooks/queries/useGetBusinessReportsQuery/useGetBusinessReportsQuery.ts

```diff
@@ -0,0 +1,15 @@
+import { useIsAuthenticated } from '@/domains/auth/context/AuthProvider/hooks/useIsAuthenticated/useIsAuthenticated';
+import { businessReportsQueryKey } from '@/domains/business-reports/query-keys';
+import { useQuery } from '@tanstack/react-query';
+
+export const useGetBusinessReportsQuery = ({ businessId }: { businessId: string }) => {
+  const isAuthenticated = useIsAuthenticated();
+
+  console.log('QUERY');
+
+  return useQuery({
+    ...businessReportsQueryKey.ongoingReportsList({ businessId }),
```

Comment:
Leading with `list` will be also more autocomplete friendly.
```suggestion
    ...businessReportsQueryKey.listOngoingReports({ businessId }),
```